dtfs-central-api/api-tests/v1/routes/middleware/headers/checkApiKey.api-test.js

```diff
@@ -1,5 +1,5 @@
 const dotenv = require('dotenv');
-const checkApiKey = require('../../../../../src/v1/routes/middleware/headers/checkApiKey');
+const checkApiKey = require('../../../../../src/v1/routes/middleware/headers/check-api-key');
```

Comment:
Nitpick as not yours but if this is check-api-key the file name should also be check-api-key.api-test.js

Reply:
fixed in my PR